core/sqf/sql/scripts/install_local_hadoop

```diff
@@ -602,15 +612,24 @@ MYSQL_TAR=mysql-5.6.10-linux-glibc2.5-x86_64.tar.gz
 MYSQL_JDBC_URL=http://cdn.mysql.com/archives/mysql-connector-java-5.1
 MYSQL_JDBC_TAR=mysql-connector-java-5.1.23.tar.gz
 
-HIVE_MIRROR_URL=https://archive.apache.org/dist/hive/hive-0.13.1
-HIVE_PREFIX=apache-hive-0.13.1-bin
+HIVE_MIRROR_URL=http://archive.cloudera.com/cdh5/cdh/5
+HIVE_PREFIX=hive-1.1.0-cdh5.4.4
 HIVE_TAR=${HIVE_PREFIX}.tar.gz
 
+if [[ "$HBASE_DISTRO" = "APACHE" ]]; then
+    HIVE_PREFIX=apache-hive-1.2.0-bin
+    HIVE_TAR=${HIVE_PREFIX}.tar.gz
+fi
+
+
 HBASE_MIRROR_URL=http://archive.cloudera.com/cdh5/cdh/5
+HBASE_TAR=hbase-1.0.0-cdh5.4.4.tar.gz
 
-HBASE_TAR=hbase-0.98.6-cdh5.3.0.tar.gz
-if [[ "$SQ_HBASE_DISTRO" = "HDP" ]]; then
-    HBASE_TAR=hbase-0.98.4.2.2.0.0-2041-hadoop2.tar.gz
+if [[ "$HBASE_DISTRO" = "HDP" ]]; then
+    HBASE_TAR=hbase-1.1.2.2.3.2.0-2950.tar.gz
```

Comment:
Same here, also set HBASE_MIRROR_URL? You may not see this as a problem if you are using cached objects.